interface_test.go

```diff
@@ -11,7 +11,7 @@ func TestTableCanBeSerializedAsJson(t *testing.T) {
 		PartitionKey:          []FieldName{"bar"},
 		PrimaryKey:            []FieldName{"bar"},
 		Values:                []FieldName{"baz"},
-		Types:                 map[FieldName]FieldType{"bar": "string", "baz": "int"},
+		Types:                 FieldTypesByName{"bar": "string", "baz": "int"},
```

Comment:
Is this an implicit way to initialize a map in go?

Reply:
I just replaced the unnamed type by its named type.